draft-ietf-trans-rfc6962-bis.md

```diff
@@ -495,6 +495,9 @@ received a consistency proof between the two (e.g., in a `TransItem` of type
 `consistency_proof_v2`), the following algorithm may be used to verify the
 consistency proof:
 
+1. If `consistency_path` is an empty array, stop and fail the proof
+   verification.
+
 1. If `first` is an exact power of 2, then prepend `first_hash` to the
```

Comment:
Now there are two step 1s.  Please renumber the list of steps.

Reply:
markdown renumbers numbered lists so this is not necessary.  to avoid confusion I renumbered them all to be "1." :)

Reply:
I see 1, 1, 2, 3, 4, 5, 6 at the moment.

Reply:
I would much prefer the numbering in the .md source to be correct, so that anyone who reads the .md source in the future does not get confused.

Reply:
fine, changed and pushed.